src/views/Dev.vue

```diff
@@ -0,0 +1,42 @@
+<template lang="pug">
+div
+  h3 Developer zone
+  | Just some tools to aid in development and debugging.
+
+  div
+    b-btn(@click="query_window_timing()") Run window query
+    b-btn(@click="query_browser_timing()") Run browser query
+</template>
+
+<script>
+import moment from 'moment';
+
+console.log(moment);
+
+export default {
+  name: "aw-dev",
+
+  data: function() {
+    return {
+      queryOptions: {
+        aw_client: this.$aw,
+        date: moment().format('YYYY-MM-DD'),
+        host: 'erb-laptop2-arch',
```

Comment:
If you want this to go into master you should at least stop hardcoding this.
Just adding a text field or something as an input though should be good enough as i assume it's made to be a rough tool rather than for user debugging.

Reply:
Ah right, I forgot. Will do.